cmd/edv-rest/main.go

```diff
@@ -7,13 +7,14 @@ SPDX-License-Identifier: Apache-2.0
 package main
 
 import (
-	"log"
-
 	"github.com/spf13/cobra"
+	"github.com/trustbloc/edge-core/pkg/log"
 
 	"github.com/trustbloc/edv/cmd/edv-rest/startcmd"
 )
 
+var logger = log.New("edv/cmd/edv-rest")
```

Comment:
edv-rest should be sufficient

Reply:
Done

Reply:
I also renamed the log module in start.go to be edv-rest as well. They are both packages under the edv-rest folder, but they are separate packages. Question: is it a problem if multiple packages have the same module name? @sandrask @sudeshrshetty